src/Iris/IO.hs

```diff
@@ -0,0 +1,95 @@
+{- |
+Module                  : Iris.IO
+Copyright               : (c) 2022 Dmitrii Kovanikov
+SPDX-License-Identifier : MPL-2.0
+Maintainer              : Dmitrii Kovanikov <kovanikov@gmail.com>
+Stability               : Experimental
+Portability             : Portable
+
+Functions for IO, such as writing Text to stdout and stderr.
+
+
+Usage example:
+
+@
+import qualified Iris
+
+main = do
+    Iris.outLn "This goes to stdout"
+    Iris.errLn "This goes to stderr"
+@
+
+Results in:
+
+@
+\$ ./app
+This goes to stdout
+This goes to stderr
+@
+
+@since x.x.x.x
+-}
+module Iris.IO (
+    out,
+    outLn,
+    err,
+    errLn,
+) where
+
+import Data.Text (Text)
+import qualified Data.Text.IO as Text
+import System.IO (stderr, stdout)
+
+{- | Write the given Text to stdout.
+No linefeed at the end.
+
+@
+ghci> do Iris.out "foo" >> Iris.out "bar"
+foobarghci>
+
+@
+@since x.x.x.x
+-}
+out :: Text -> IO ()
```

Comment:
Let's actually make this function work in a polymoprhic `MonadIO`, so people won't need to write `liftIO`. And you actually remove `liftIO $ ...` in all places where you use `out`, `outLn` and etc.

```suggestion
out :: MonadIO m => Text -> m ()
```